examples/security/keycloak/kafka-authz-realm.json

```diff
@@ -0,0 +1,652 @@
+{
```

Comment:
Should the directory really be named `keycloak` and not `oauth`? It is example of OAuth authentication, or?

Reply:
It is an example of 'keycloak' authorization. I think using 'keycloak' best describes the content, which is a keycloak realm export JSON.

Reply:
Ok, good.

Reply:
Actually, should we call it `keycloak-authorization` maybe then?